src/main/java/de/hysky/skyblocker/utils/render/RenderHelper.java

```diff
@@ -60,6 +61,20 @@ public static void renderFilledWithBeaconBeam(WorldRenderContext context, BlockP
         renderFilled(context, pos, colorComponents, alpha, throughWalls);
         renderBeaconBeam(context, pos, colorComponents);
     }
+    public static void renderFilled(WorldRenderContext context, Box boundingBox, float[] colorComponents, float alpha, boolean throughWalls) {
```

Comment:
This could just call `renderFilled(WorldRenderContext context, Vec3d pos, Vec3d dimensions, float[] colorComponents, float alpha, boolean throughWalls)` (the last `renderFilled` method). This currently skips culling.